### PR TITLE
Update scarb installation to use asdf

### PR DESCRIPTION
**Pull request by amanusk**
### Description of the Changes

Updating the env setup instructions to use asdf and not scarb script

If the PR is related to an open issue, please also link to it.

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Please paste the specific URL(s) of the content that this PR addresses here.

https://starknet-io.github.io/starknet-docs/pr-1398/quick-start/environment-setup/

### Check List

- [X] Changes have been done against main branch, and PR does not conflict
- [X] PR title is meaningful, e.g: `fix: minor typos in README`

### CODE REVIEW

File: components/Starknet/modules/quick-start/pages/environment-setup.adoc [hunk 1]
```diff
@@ -1 +1,152 @@
...
+== Installing Scarb
+
+Scarb is compatible with macOS, Linux, and Windows operating systems.
+
+
+=== MacOS and Linux installation
+
+It is highly recommended to install Scarb via the link:https://asdf-vm.com/[asdf] version manager.
+
+Installing Scarb with `asdf` seamlessly integrates with the Cairo VSCode extension, and allows to easily switch between different versions of Scarb, globally or per project.
```
Comment (LandauRaz) [inline components/Starknet/modules/quick-start/pages/environment-setup.adoc:92]: Please be consistent and use either asfd or `asfs` throughout the entire subsection

Comment (LandauRaz) [inline components/Starknet/modules/quick-start/pages/environment-setup.adoc:92]: Please add a link to the "Visual Studio Code Extension" page in the "Tools and resources" section

**Review by LandauRaz — CHANGES_REQUESTED** [review]

File: components/Starknet/modules/quick-start/pages/environment-setup.adoc [hunk 2]
```diff
@@ -1 +1,215 @@
...
+
+=== MacOS and Linux installation
+
+It is highly recommended to install Scarb via the link:https://asdf-vm.com/[asdf] version manager.
+
+Installing Scarb with `asdf` seamlessly integrates with the Cairo xref:tools:devtools/vscode.adoc[VSCode extension], and allows to easily switch between different versions of Scarb, globally or per project.
+
+.Procedure
+
+. Follow the installation steps of `asdf`` on the link:https://asdf-vm.com/guide/getting-started.html[official website].
```
Comment (LandauRaz) [inline components/Starknet/modules/quick-start/pages/environment-setup.adoc:101]: `asdf`` should be `asdf`

File: components/Starknet/modules/quick-start/pages/environment-setup.adoc [hunk 3]
```diff
@@ -1 +1,215 @@
...
+[source, bash]
+----
+scarb --version
+----
+
+You should see the version of Scarb, Cairo and Sierra.
+
+[TIP]
+====
+See additional commands for updating Scarb and switching between versions in the link:https://asdf-vm.com/manage/commands.html[asdf documentation], or by running `asdf --help`.
```
Comment (LandauRaz) [inline components/Starknet/modules/quick-start/pages/environment-setup.adoc:138]: [asdf documentation] should be [`asdf` documentation]

File: components/Starknet/modules/quick-start/pages/environment-setup.adoc [hunk 4]
```diff
@@ -1 +1,215 @@
...
+|A command-line interface that allows you to interact with Starknet.
+|https://book.starkli.rs/[book.starkli.rs]
+|https://github.com/xJonathanLEI/starkli[github.com/xJonathanLEI/starkli]
+
+|Scarb
+|A build toolchain and package manager for Cairo and Starknet ecosystems.
+|https://docs.swmansion.com/scarb/[docs.swmansion.com/scarb]
+|https://github.com/software-mansion/scarb[github.com/software-mansion/scarb]
+
+|Starknet Foundry
```
Comment (LandauRaz) [inline components/Starknet/modules/quick-start/pages/environment-setup.adoc:23]: Please open a separate PR for changes regarding Starknet Foundry

**Review by LandauRaz — APPROVED** [review]

**Comment by chadoh** [comment]
This change undid https://github.com/starknet-io/starknet-docs/pull/876 by @stoobie, which switched this repo to use an installation guide from [docs-common-content](https://github.com/starknet-io/docs-common-content). Was that intentional? If I want to make future changes to the installation guide, should I make them in this repository, or in `docs-common-content`?

**Comment by LandauRaz** [comment]
Hey @chadoh, `docs-common-content` is now deprecated so all changes should be done in this repository.

**Pull request merged**